packages/design-system/src/components/tooltip/index.js

```diff
@@ -177,7 +177,7 @@ function Tooltip({
   const positionPlacement = useCallback(
     ({ offset }) => {
       // check to see if there's an overlap with the window's bottom edge
-      const neededVerticalSpace = offset.bottom;
+      const neededVerticalSpace = offset.y + offset.popupHeight + TAIL_HEIGHT;
```

Comment:
Tail height is needed regardless of whether `hasTail` is true or not because the popup is placed in the same spot, ie 8 px below the anchor.